demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -335,3 +337,38 @@ def is_there_spaces_in_the_end_of_id(self):
                 return False
 
         return True
+
+    def validate_readme_exists(self, validate_all: bool):
+        """
+            Validates if there is a readme file in the same folder as the caller file.
+            The validation is processed only on added or modified files.
+
+            Args:
+                validate_all: (bool) is the validation being run with -a
+            Return:
+               True if the readme file exits False with an error otherwise
+        """
+        if validate_all:
+            return True
+        file_path = os.path.normpath(self.file_path)
+        path_split = file_path.split(os.sep)
+        file_type = self.structure_validator.file_type
+        if file_type == PLAYBOOK:
+            to_replace = os.path.splitext(path_split[-1])[-1]
+            readme_path = file_path.replace(to_replace, '_README.md')
+        elif file_type == SCRIPT or INTEGRATION:
```

Comment:
It always returned True, Why didn't the tests catch it?
```suggestion
        elif file_type in {SCRIPT, INTEGRATION}:
```

Reply:
it didnt catch it because playbook was checked before the script and integration so didnt enter the else with a playbook. and for script and integration this is the correct test.
other classes dont call this method so it was not catched in the tests. 
i can maybe add a new test trying to enter a test playbook which should result in an True.